cargo-dist/src/tasks.rs

```diff
@@ -889,8 +889,12 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
         for (pkg_idx, binary_name) in bins.clone() {
             let package = self.workspace.package(pkg_idx);
             let package_metadata = self.package_metadata(pkg_idx);
-            let version = package.version.as_ref().unwrap().cargo();
-            let pkg_id = package.cargo_package_id.clone().unwrap();
+            let version = package
+                .version
+                .as_ref()
+                .expect("Package version is mandatory!")
+                .cargo();
```

Comment:
Just checking you're aware that `.cargo()` will also panic if there's a Not Rust Project?

https://docs.rs/axoproject/latest/src/axoproject/lib.rs.html#337-344

Reply:
Yeah, I'll be dealing with that in other branches.